commands/warehousecmd.py

```diff
@@ -0,0 +1,6 @@
+from commands import warehouse
+
+
+def warehousecmd(update, context):
```

Comment:
should live in warehouse.py

Reply:
I was thinking it makes more sense to keep everything that's a command function in `commands`, but if you would prefer it being in warehouse that's fine

Reply:
oh!!!!!! wait I just realized what you meant. Yeah got it

Reply:
Resolved in 24e2cb9